jablib/src/test/java/org/jabref/logic/citation/SearchCitationsRelationsServiceTest.java

```diff
@@ -157,7 +167,7 @@ void serviceShouldCallTheFetcherForReferencesWhenRepositoryIsUpdatable() {
             BibEntry newReference = new BibEntry();
             List<BibEntry> referencesToReturn = List.of(newReference);
             Map<BibEntry, List<BibEntry>> referencesDatabase = new HashMap<>();
-            CitationFetcher fetcher = createMockFetcher(referencer, null, referencesToReturn);
+            CitationFetcher fetcher = createMockFetcher(referencer, null, referencesToReturn, null);
```

Comment:
you are passing `null`

If it is too complicated with non-null (line 27), just keep using null. This is OK. - there is no optional any more in line 27